tests/test_json.py

```diff
@@ -204,6 +204,37 @@ class Model(BaseModel):
     assert Model(__root__=['a', 'b']).json() == '["a", "b"]'
 
 
+def test_encode_nested_root():
```

Comment:
this has nothing to do with `.json()` in particular, it should be true of `.dict()` too, so tests shouldn't be in this file.

Reply:
I am not sure to understand your comment. Those tests build models with nested root as child elements, then calls the json method and the expected result is a json without the `__root__` intermediate key.
(The behavior change concerns only the json method, doing it in dict method is breaking other tests)